src/schema.graphql

```
@@ -10,6 +10,7 @@ type Query {
   hello: String
   user: User
   users(uids: [ID!]!): [User!]!
+  getUser(email: String!): User
```

You could make the returned User non-nullable and throw an error instead of returning undefined.
Otherwise I'd return null explicitly, as just returning nothing is more of a "stop execution here" type of thing instead of clearly "I want to return nothing here". If that makes sense :D

---

Updated, now there will always be an Error thrown if no user can't be returned and the return types are non-nullable.